app/src/main/java/com/pvp/app/ui/screen/task/TextFieldWithErrors.kt

```diff
@@ -0,0 +1,85 @@
+package com.yourapp.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Error
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.KeyboardType
+
+
+/**
+ * @param value text field input text
+ * @param onValueChange (optional) callback triggered when text inside text field is updated
+ * @param validationPolicies (optional) how text inside should be validated,
+ *          should be specified via InputValidator
+ * @param label label displayed inside text container
+ * @param keyboardOptions (optional) software keyboard options that contains configuration such as KeyboardType and ImeAction.
+ * @param modifier (optional) the Modifier applies to the text field
+ */
+@Composable
+fun TextFieldWithErrors(
```

Comment:
This possibly could be simplified to a `TextField`. As the component supports errors, but they are optional, developer can use this component without error handling too. About more suited naming for these components we should discuss in the PR comment section.